src/drive/reducers/index.js

```diff
@@ -9,8 +9,12 @@ import renameReducer from '../ducks/files/rename'
 import upload from '../ducks/upload'
 import alerterReducer from 'cozy-ui/react/Alerter'
 import availableOffline from '../ducks/files/availableOffline'
+import { reducer } from 'redux-cozy-client'
```

Comment:
Well, a lot of exported things are fairly important in `redux-cozy-client`, this is not simply a duck. That's why I don't see why the reducer should be the default instead of the middleware or the `cozyConnect` HOC.